gramhopper/configuration/trigger_response_params.py

```diff
@@ -0,0 +1,29 @@
+from dataclasses import dataclass
+from typing import Dict, Type
+from .trigger_response import TriggerResponse
+from .triggers_reponses_parsers import BaseParser, TriggerParser, ResponseParser
+from ..triggers.basic_triggers import BaseTrigger
+from ..responses.basic_responses import BaseResponse
+
+
+@dataclass
+class TriggerResponseParams:
+    singular_key: str
+    globals: Dict[str, TriggerResponse]
+    parser: Type[BaseParser]
+
+    @property
+    def plural_key(self):
```

Comment:
OK, I need to see an example for this.

Reply:
For example, in a rule configuration, the triggers' key is `trigger`, but the global triggers' key is `triggers`.
Same for `response` and `responses`.